### ISSUE TITLE
Create alternate "package" types for this repo

### ISSUE THREAD

**Issue by bnvk** [simonv3/factbook-country-data#2]
Cool stuff here! Would be good to have this as a `npm` and `datapackage` format as well as bower!

**Comment by simonv3** [comment]
Yep yep, for sure! Since this is just a manually collected dump it'd be great if we could first automate that gathering somehow.
